framework/test/jest/specs/unit/controller/schema/__snapshots__/constants.spec.js.snap

```diff
@@ -412,70 +87,5 @@ Object {
     ],
     "type": "object",
   },
-  "transactionTypes": Object {
-    "additionalProperties": false,
-    "id": "transactionTypes",
-    "properties": Object {
-      "DAPP": Object {
-        "enum": Array [
-          5,
-        ],
-        "type": "integer",
-      },
-      "DELEGATE": Object {
-        "enum": Array [
-          2,
-        ],
-        "type": "integer",
-      },
-      "IN_TRANSFER": Object {
-        "enum": Array [
-          6,
-        ],
-        "type": "integer",
-      },
-      "MULTI": Object {
-        "enum": Array [
-          4,
-        ],
-        "type": "integer",
-      },
-      "OUT_TRANSFER": Object {
-        "enum": Array [
-          7,
-        ],
-        "type": "integer",
-      },
-      "SEND": Object {
-        "enum": Array [
-          0,
-        ],
-        "type": "integer",
-      },
-      "SIGNATURE": Object {
-        "enum": Array [
-          1,
-        ],
-        "type": "integer",
-      },
-      "VOTE": Object {
-        "enum": Array [
-          3,
-        ],
-        "type": "integer",
-      },
-    },
-    "required": Array [
-      "SEND",
-      "SIGNATURE",
-      "DELEGATE",
-      "VOTE",
-      "MULTI",
-      "DAPP",
-      "IN_TRANSFER",
-      "OUT_TRANSFER",
-    ],
-    "type": "object",
-  },
```

Comment:
Just thought that for the sake for more safety we should not remove the schema even though these are hard coded values now. But still this schema validation does not harm.